README.md

```diff
@@ -11,13 +11,12 @@ shell.
 ## Documentation
 
 The documentation can be found at
-[http://doc.metashell.org](http://doc.metashell.org)
+[http://metashell.org](http://metashell.org)
 
 ## Current build status
 
 Service | Status
 --- | ---
-[doc.metashell.org](http://doc.metashell.org/) | [![Documentation Status](https://readthedocs.org/projects/metashell/badge/?version=latest "Documentation Status")](http://metashell.readthedocs.org/en/latest/?badge=latest) |
```

Comment:
Because the launcher of the demo has been integrated into the documentation, which will be served by the demo server in the future (not by readthedocs).